app/models/classification.rb

```diff
@@ -46,6 +46,27 @@ class Classification < ApplicationRecord
 
   FIXTURE_FILE = FIXTURE_DIR.join("classifications.yml")
 
+  def self.parent_classifications
+    where(:classifications => {:parent_id => 0})
```

Comment:
This will change from 0 to nil in #18301.  Consider abstracting this using `is_category` scope.